ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -258,6 +258,7 @@
 				5340828A26CFF298007716E1 /* Recovered References */,
 			);
 			sourceTree = "<group>";
+			usesTabs = 0;
```

Comment:
Why is this change relevant to this PR?

Reply:
sorry, maybe it's auto updated by my local Apex config, reverted